src/test/java/EnergyTest.java

```diff
@@ -0,0 +1,38 @@
+import imageData.Energy;
```

Comment:
The package of a test class should match the package of the class you are testing.